shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/profile/orders/presentation/OrderHistoryScreen.kt

```diff
@@ -226,122 +222,6 @@ fun OrderListSection(
-                            ) {
-                                PlantImage(
-                                    url = plant.image,
-                                    modifier = Modifier
-                                        .width(imageWidth)
-                                        .fillMaxHeight()
-                                )
-                            }
-                        }
-                    }
-                }
-
-                Column(
-                    modifier = Modifier
-                        .padding(start = 8.dp)
-                ) {
-                    Text(
-                        text = itemState.orderId.asString(),
-                        style = Typography.titleSmall,
-                        fontStyle = FontStyle.Italic,
-                        fontWeight = FontWeight.Light,
-                        color = Color.Black.copy(alpha = 0.6f),
-                    )
-                    Text(
-                        text = itemState.title.asString(),
-                        style = Typography.titleMedium,
-                        maxLines = 2,
-                        overflow = TextOverflow.Ellipsis,
-                        color = Color.Black.copy(alpha = 0.6f),
-                        modifier = Modifier.height(50.dp)
-                    )
-                    Spacer(Modifier.height(4.dp))
-                    Text(
-                        text = itemState.subTitle.asString(),
-                        style = Typography.titleSmall,
-                        color = Color.Gray.copy(alpha = 0.9f),
-                    )
-                    Spacer(Modifier.height(6.dp))
-                    Text(
-                        text = itemState.totalPrice.asString(),
-                        style = Typography.titleMedium,
-                        color = Color.Black.copy(alpha = 0.6f),
-                    )
-                }
-            }
-            Spacer(Modifier.height(16.dp))
-            Row(
-                modifier = Modifier
-                    .fillMaxWidth(),
-                horizontalArrangement = Arrangement.SpaceBetween
-            ) {
-                Button(
-                    onClick = { onEvent(OrderHistoryEvents.PrimaryButtonPressed(itemState.data)) },
-                    modifier = Modifier.fillMaxWidth().weight(1f)
-                ) {
-                    Text(
-                        text = itemState.primaryButtonTitle.asString()
-                    )
-                }
-
-                Spacer(Modifier.width(10.dp))
-                Button(
-                    onClick = { onEvent(OrderHistoryEvents.SecondaryButtonPressed(itemState.data)) },
-                    modifier = Modifier.fillMaxWidth().weight(1f)
-                ) {
-                    Text(
-                        text = itemState.secondaryButtonTitle.asString()
-                    )
-                }
-            }
-        }
-    }
-}
-
 @Composable
 fun EmptyMessage(
```

Comment:
I think it is not being used anywhere else. Could this be private? Also same for the OrderListSection?